lib/matplotlib/collections.py

```diff
@@ -788,6 +788,7 @@ def update_from(self, other):
         self._facecolors = other._facecolors
         self._linewidths = other._linewidths
         self._linestyles = other._linestyles
+        self._us_linestyles = other._us_linestyles
```

Comment:
Is there anywhere that documents that 'us' means unscaled? It's rather an opaque prefix to me.